test/e2e/live/rebuild-hermes.test.ts

```
@@ -814,38 +1023,70 @@ test(STALE_BASE_REBUILD
       await artifacts.writeText(
-        "phase-5-current-base-note.txt",
-        "Current Hermes base is constructed once by the authoritative rebuild in phase 6; the redundant setup build was removed. (#7144)\n",
+        "phase-5-current-base-reuse.txt",
+        `Reusing phase 1 Hermes base ${phase1BaseResolution.ref} (${phase1BaseResolution.digest ?? phase1BaseResolution.imageId}) through ${CURRENT_BASE_REUSE_TAG}; rebuild must not construct it again.\n`,
       );
       break;
+    }
     case true:
       progress.phase("phase 5 stale base setup");
+      const classification =
+        staleBaseClassification ?? fail("stale rebuild lane did not classify its old base hint");
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)